packages/back-end/src/controllers/features.ts

```diff
@@ -520,20 +527,106 @@ export async function postFeatureRebase(
     status: 200,
   });
 }
+export async function postFeatureRequestReview(
+  req: AuthRequest<
+    {
+      comment: string;
+    },
+    { id: string; version: string }
+  >,
+  res: Response
+) {
+  const context = getContextFromReq(req);
+  const { id, version } = req.params;
+  const { comment } = req.body;
+  const feature = await getFeature(context, id);
+  if (!feature) {
+    throw new Error("Could not find feature");
+  }
+
+  const revision = await getRevision(
+    context.org.id,
+    feature.id,
+    parseInt(version)
+  );
+  if (!revision) {
+    throw new Error("Could not find feature revision");
+  }
+  if (revision.status !== "draft") {
+    throw new Error("Can only request review if is a draft");
+  }
+  await markRevisionAsReviewRequested(revision, res.locals.eventAudit, comment);
+  res.status(200).json({
+    status: 200,
+  });
+}
+
+export async function postFeatureReviewOrComment(
+  req: AuthRequest<
+    {
+      comment: string;
+      review?: ReviewSubmittedType;
+    },
+    { id: string; version: string }
+  >,
+  res: Response
+) {
+  const context = getContextFromReq(req);
+  const { id, version } = req.params;
+  const { comment, review = "Comment" } = req.body;
+  const feature = await getFeature(context, id);
+  if (!feature) {
+    throw new Error("Could not find feature");
+  }
+  req.checkPermissions("canReview");
```

Comment:
This permission check should be project-scoped depending on what project the feature is in.

Reply:
do you have an example of a project scoped?